tests/expected/abort/main.rs

```diff
@@ -14,7 +14,7 @@ fn main() {
         }
         if i == 2 {
             // This should never happen.
-            process::abort();
+            process::exit(1);
```

Comment:
The comment says test `abort` but now we're calling `exit`. Is there a reason for choosing either?

Reply:
I just wanted to increase test coverage